src/types/ErrorAction.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorAction {
+  EMPTY = 'empty_title',
+  LOAD = 'load',
+  ADD = 'add',
+  DELETE = 'delete',
+  UPDATE = 'update',
```

Comment:
```suggestion
export enum ErrorAction {
  Empty = 'empty_title',
  Load = 'load',
  Add = 'add',
  Delete = 'delete',
  Update = 'update',
```
Enum properties should be in PascalCase, not upper case